webpack.config.js

```diff
@@ -113,7 +113,8 @@ module.exports = {
     alias: {
       css: path.join(__dirname, "/troposphere/static/css/"),
       images: path.join(__dirname, "/troposphere/static/images/"),
-      highcharts: "highcharts-commonjs"
+      highcharts: "highcharts-commonjs",
+      react: path.join(__dirname, "/node_modules/react")
```

Comment:
This PR was building and rendering some components from Troposphere-ui so it looked like it was working. However, components that do things like attach refs to DOM elements expose an error caused by having multiple versions of react. 

This alias insures that any reference to react resolves to this path so that webpack doesn't try to bundle components with different versions of React.